demos/common/cpp_gapi/utils_gapi/src/stream_source.cpp

```diff
@@ -61,6 +61,8 @@ cv::MediaFrame::View MediaBGRAdapter::access(cv::MediaFrame::Access) {
 bool MediaCommonCapSrc::pull(cv::gapi::wip::Data& data) {
```

Comment:
do we have source that returns `cv::Mat`?

Reply:
yes, it's `CommonSrc` which will require add timestamp to that instead of MediaCap

Reply:
You'd proceed with `cv::GMat` so far...